metrics.md

```diff
@@ -0,0 +1,238 @@
+# Sync Metrics
+
+## Goal
+
+Metrics help us understand how people are using our products which in turn helps us improve them in ways that increase the value folks get out of them. We aren't interested in using metrics to turn data about our users into a product that we can sell, we just want to provide the best possible user experience.
+
+## Artifacts
+
+Engagement and retention are our bread and butter metrics. By improving these measurements we can be confident that we're improving the user experience.
+
+### Engagement Ratio
+
+    Of all the people who used Sync in the last (28) days, what percentage of them used it on a given day?
+
+Having a high ratio of daily users in relation to monthly users  indicates that a majority of users are using Sync frequently. Tracking the engagement ratio allows us to monitor how changes to the user experience affect how frequently users engage.
```

Comment:
It might help to add an example. 

> For example, if we calculate an engagement ratio of 0.5, it means that users return to our product every 2 days. The calculation would look like 14 days / 28 days.
> Inversely, if users engage with our product on average 7 days per month, we will have an engagement ration of 0.25 (7/28).